src/tests/unit/components/filter-input/FilterInput.spec.jsx

```
@@ -1,27 +1,45 @@
     const { getByTestId } = render(
-      <FilterInput onChange={handleChange} value='test' />
+      <FilterInput value='hello' onChange={handleChange} />
     )
     const clearButton = getByTestId('clear-button')
     fireEvent.click(clearButton)
     expect(handleChange).toHaveBeenCalledWith('')
   })
+
+  it('calls updateFilter function on search icon (if used as button)', () => {
+    const handleChange = vi.fn()
+    render(<FilterInput value='' onChange={handleChange} />)
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)